src/main/java/org/folio/event/handler/InventoryCreateAsyncRecordHandler.java

```diff
@@ -86,9 +91,17 @@ private void verifyKafkaRecord(KafkaConsumerRecord<String, String> kafkaConsumer
   private Future<String> getCentralTenantId(Map<String, String> headers) {
     var requestContext = new RequestContext(getContext(), headers);
     return consortiumConfigurationService.getConsortiumConfiguration(requestContext)
-      .map(optionalConsortiumConfiguration -> optionalConsortiumConfiguration
-        .map(ConsortiumConfiguration::centralTenantId)
-        .orElse(null));
+      .compose(consortiumConfiguration -> {
+        if (consortiumConfiguration.isEmpty()) {
+          return Future.succeededFuture();
+        }
+        var configuration = consortiumConfiguration.get();
+        return settingService.getSettingByKey(SettingKey.CENTRAL_ORDERING_ENABLED, headers, getContext())
```

Comment:
we also need to use cache here, this setting never changes, so by using cache we can reduce lot of calls to DB and so these sql connections from pool can be used for other important work

Reply:
Added cache here aswell